test/specs/utils/helpers.js

```diff
@@ -268,6 +268,72 @@ describe('utils helpers', () => {
 			);
 		});
 	});
+	describe('#normalizeAmount', () => {
+		Given(
+			'an amount "120" with normalized amount "12000000000"',
+			given.anAmountWithNormalizedAmount,
+			() => {
+				When(
+					'normalizeAmount is called on the amount',
+					when.normalizeAmountIsCalledOnTheAmount,
+					() => {
+						Then(
+							'it should return the normalized amount',
+							then.itShouldReturnTheNormalizedAmount,
+						);
+					},
+				);
+			},
+		);
+		Given(
+			'an amount "123.456" with normalized amount "12345600000"',
+			given.anAmountWithNormalizedAmount,
+			() => {
+				When(
+					'normalizeAmount is called on the amount',
+					when.normalizeAmountIsCalledOnTheAmount,
+					() => {
+						Then(
+							'it should return the normalized amount',
+							then.itShouldReturnTheNormalizedAmount,
+						);
+					},
+				);
+			},
+		);
+		Given(
+			'an amount "0" with normalized amount "0"',
+			given.anAmountWithNormalizedAmount,
+			() => {
+				When(
+					'normalizeAmount is called on the amount',
+					when.normalizeAmountIsCalledOnTheAmount,
+					() => {
+						Then(
+							'it should return the normalized amount',
+							then.itShouldReturnTheNormalizedAmount,
+						);
+					},
+				);
+			},
+		);
+		Given(
```

Comment:
It might be good to add edge cases where we have decimal places more than 8

Reply:
I thought about calling `validateAmount` in here, but that would seem to change the locus of responsibility: right now it's in the action, along with validating the address.

Reply:
not necessarily about `validAmount`, but as for `unit test` I think it's good to expect error thrown for when length of `postArray` is too long.
```
const pad = new Array(DECIMAL_PLACES - postArray.length).fill('0');
```